internal/controller/bucket/policy.go

```diff
@@ -0,0 +1,203 @@
+package bucket
+
+import (
+	"context"
+
+	"github.com/aws/aws-sdk-go-v2/aws"
+	"github.com/aws/smithy-go"
+
+	"github.com/crossplane/crossplane-runtime/pkg/errors"
+	"github.com/crossplane/crossplane-runtime/pkg/logging"
+
+	"github.com/linode/provider-ceph/apis/provider-ceph/v1alpha1"
+	apisv1alpha1 "github.com/linode/provider-ceph/apis/v1alpha1"
+	"github.com/linode/provider-ceph/internal/backendstore"
+	"github.com/linode/provider-ceph/internal/consts"
+	"github.com/linode/provider-ceph/internal/controller/s3clienthandler"
+	"github.com/linode/provider-ceph/internal/otel/traces"
+	"github.com/linode/provider-ceph/internal/rgw"
+	"go.opentelemetry.io/otel"
+)
+
+// PolicyClient is the client for API methods and reconciling the BucketPolicy
+type BucketPolicyClient struct {
+	backendStore    *backendstore.BackendStore
+	s3ClientHandler *s3clienthandler.Handler
+	log             logging.Logger
+}
+
+// NewBucketPolicyClient creates the client for Accelerate Configuration
```

Comment:
I'm not sure what `Accelerate Configuration` is, I'm guessing it was copy/pasted from `lifecycleconfiguration.go`? Not sure why it's there either :sweat_smile:  it should be removed from both files.